docs/sources/alerting/unified-alerting/high-availability.md

```diff
@@ -0,0 +1,64 @@
+
+```
+  .─────.
+ ╱       ╲                                                                      ┌────────────────┐
+(  User   )──────┐                        ┌──────────────────────────────────┐  │                │
+ `.     ,'       │                        │┌─────────┐      ┌──────────────┐ │  │                ▼
+   `───'         │                        ││Scheduler│──────▶Alertmananager│─┼──┘    ┌──────────────────────┐
+                 │      ┌───────────┐  ┌─▶│└─────────┘      ▲──────────────┤ │       │                      │
+  .─────.        │      │   Load    │  │  │Grafana          │              │ │       │                      │
+ ╱       ╲       │      │ Balancing │  │  └─────────────────┼──────────────┼─┘       │     Integrations     │
+(  User   )──────┼─────▶│  Reverse  │──┤  ┌─────────────────┼──────────────┼─┐       │                      │
+ `.     ,'       │      │   Proxy   │  │  │┌─────────┐      ├──────────────▼ │       │                      │
+   `───'         │      └───────────┘  │  ││Scheduler│──────▶Alertmananager│─┼──┐    └──────────────────────┘
+                 │                     └─▶│└─────────┘      └──────────────┘ │  │                ▲
+  .─────.        │                        │Grafana                           │  │                │
+ ╱       ╲       │                        └──────────────────────────────────┘  └────────────────┘
+(  User   )──────┘
+ `.     ,'
+   `───'
+```
+
+The coordination between Grafana instances happens via [Gossip](https://en.wikipedia.org/wiki/Gossip_protocol). Alerts are not gossiped between instances. It is expected that each scheduler delivers the same alerts to each Alertmanager.
+
+The two types of messages that are gossiped between instances are:
+
+- Notification logs: Who (which instance) notified what (which alert)
+- Silences: If an alert should fire or not
+
+These two states are persisted in the database periodically and when Grafana is gracefully shutdown.
+
+## Setup
+
+To enable high availability support, you need to set the configuration [option `ha_peers` within the `[unified_alerting]` section]({{< relref "../../administration/configuration.md#unified_alerting" >}}). Setting this option to at least 1 peer will trigger the high availability mode for Grafana 8 Alerts. Communication happens using port 9094 on both UDP and TCP, so please make sure each instance has access on these.
+
+The idea here is that you:
+
+- Set `ha_peers` to the set of hosts for each grafana instance in the cluster (using a format of `host:port`) e.g. `ha_peers=10.0.0.5:9094,10.0.0.6:9094,10.0.0.7:9094`
+- Set `ha_listen_address` to the instance (or the [Pod's](https://kubernetes.io/docs/concepts/workloads/pods/) IP (using a format of `host:port`) in the case of using Kubernetes) by default it is set to listen all interfaces (`0.0.0.0`).
```

Comment:
```suggestion
To enable high availability support, you need to set the configuration option ha_peers within the [unified_alerting] section. Setting this option to at least 1 peer will trigger the high availability mode for Grafana 8 alerts. Any communication occurs using port 9094 on both UDP and TCP, so please make sure each instance has access to these.
- Set ha_peers to the set of hosts for each grafana instance in the cluster (using a format of host:port) e.g. ha_peers=10.0.0.5:9094,10.0.0.6:9094,10.0.0.7:9094
- Set ha_listen_address to the instance (or the Pod’s IP (using a format of host:port) in the case of using Kubernetes) by default, it is set to listen to all interfaces (0.0.0.0).
```

Reply:
Will be good if you add a procedure format. Also, please update the "/administration/configuration/#unified_alerting" section with the `ha_peers` option.

Reply:
I like the format, but I don't think what you've suggested is correct. Let me give it another go. 

Also, HA peers is already part of this section (https://github.com/grafana/grafana/blob/main/docs/sources/administration/configuration.md#ha_peers) - or am I missing something?

Reply:
WDYT now?

Reply:
Let me know if you have any more comments and I can amend as part of a separate PR.